variable/test/variable_bucket_test.cpp

```diff
@@ -110,6 +110,7 @@ TEST_F(VariableBucketTest, binary_operation_with_dense_broadcast) {
 
 TEST_F(VariableBucketTest, to_constituents) {
   auto [idx0, dim0, buf0] = VariableView(var).constituents<bucket<Variable>>();
+  (void)dim0;
```

Comment:
Is this like the `static_cast<void>()`? I'm not familiar with this syntax.

Reply:
Yes, it's the same in this case. This is just a c-style cast. Should probably change this.